### PR TITLE
[4.0] RTL: Fixing template editors text-align

### PR DESCRIPTION
**Pull request by infograf768**
### Summary of Changes
When editing template files in RTL, the text content should be aligned left.
Adding rtl overrides in `.../build/media_source/com_templates/css/admin-templates-default.css` both for Codemirror and no editor

### Testing Instructions
Edit a template file through administration.
No need to test with Persian language. Simpler in this case to set en-GB.xml to rtl
`<rtl>1</rtl>`

### Before patch
Screen Shot 2019-09-21 at 18 57 00

### After patch
Screen Shot 2019-09-21 at 18 55 19

@Harmageddon 
@richard67

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 4abca42bb14526eef1a6b78ac7b9e95977d52a36<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26369">issues.joomla.org/tracker/joomla-cms/26369</a>.</sub>

File: build/media_source/com_templates/css/admin-templates-default.css [hunk 1]
```diff
@@ -84,3 +84,18 @@
 .switcher-label-0, .switcher-label-1 {
 	white-space: nowrap;
 }
+
+/* RTL overrides for templates editing*/
+[dir=rtl] .diff-pane {
+	text-align:left;
```
Comment (Quy) [inline build/media_source/com_templates/css/admin-templates-default.css]:
```suggestion
	text-align: left;
```

Comment (Quy) [reply build/media_source/com_templates/css/admin-templates-default.css]: Repeat below.

Comment (infograf768) [reply build/media_source/com_templates/css/admin-templates-default.css]: will do tomorrow

**Comment by infograf768** [comment]
Space corrected.
While I was at it and to avoid another very small PR, I also corrected the display of the path of the edited file.

### Before
The slash is misplaced at the end of the path.

Screen Shot 2019-09-22 at 08 11 37

### After
The position of the slash is corrected

Screen Shot 2019-09-22 at 08 10 59

**Comment by Harmageddon** [comment]
I have tested this item :white_check_mark: successfully on 64732b0cd6dbc14dbdb5b86ca093a696780f4264<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26369">issues.joomla.org/tracker/joomla-cms/26369</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 64732b0cd6dbc14dbdb5b86ca093a696780f4264<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26369">issues.joomla.org/tracker/joomla-cms/26369</a>.</sub>

**Comment by infograf768** [comment]
rtc
thanks for testing<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26369">issues.joomla.org/tracker/joomla-cms/26369</a>.</sub>

**Comment by wilsonge** [comment]
I think it's probably better to do this in javascript than CSS. But right now I don't have the time to find how to do this properly in all places. So merging. If anyone has the time to try and put this into JS please do!

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**